esdb/persistent_subscriptions_test.go

```diff
@@ -0,0 +1,329 @@
+package esdb_test
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/EventStore/EventStore-Client-Go/esdb"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_CreatePersistentStreamSubscription(t *testing.T) {
```

Comment:
Style nit: the underscore in the identifier should be removed for consistency.

Reply:
That is how the consistency work in Go, `Test_[Name Here]`